corehq/apps/cleanup/deletable_doc_types.py

```diff
@@ -24,5 +24,6 @@
     'ForwardingRule-Deleted': (MAIN_DB,),
     'GlobalAppConfig': (settings.NEW_APPS_DB,),
     'ILSGatewayConfig': (MAIN_DB,),
+    'Invitation': (settings.NEW_USERS_GROUPS_DB,),
```

Comment:
https://github.com/dimagi/commcare-hq/pull/26673 has removed these `NEW_*_DB` variables, so you'll have to update this PR and any other that references them. Sorry for the extra pain, but hopefully removing the redundancy also makes things a little more straightforward

Reply:
Updated, and no worries, I am in favor of cleanup :)